src/Venturecraft/Revisionable/RevisionableTrait.php

```diff
@@ -239,6 +241,10 @@ public function postCreate()
                 'updated_at' => new \DateTime(),
             );
 
+            //Determine if there are any additional fields we'd like to add to our model contained in the config file, and
```

Comment:
I wouldn't leave comments. Try to have the code communicate. For example, `mergeAdditionalFields($revision[0])`.

Reply:
He comments the living shit out of this code. I'm ok with these since we're adding stuff he didn't write.